app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine", False):
+            raise NotVaccinatedError("All visitors must be vaccinated!!!")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Vaccine outdated!!!")
+        if not visitor.get("wearing_a_mask", False):
```

Comment:
First of all u don't need to use if here,  just use elif also key wearing_a_mask all time present but he has different values True or False. So u need rewrite this:
 elif not visitor["wearing_a_mask"]